GMAO_pFIO/tests/CMakeLists.txt

```diff
@@ -52,17 +52,18 @@ include_directories(
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/..)
 include_directories(${CMAKE_CURRENT_BINARY_DIR}/..)
 include_directories(${include_GMAO_pFIO})
+include_directories(${MAPL_SOURCE_DIR}/include)
 
 
 set(TESTO pfio_ctest_io.x)
 ecbuild_add_executable (
   TARGET ${TESTO}
   PROPERTIES EXCLUDE_FROM_ALL=TRUE
   SOURCES pfio_ctest_io.F90
-  LIBS GMAO_pFIO ${NETCDF_LIBRARIES} MPI::MPI_Fortran
+  LIBS MAPL_Shared GMAO_pFIO ${NETCDF_LIBRARIES} MPI::MPI_Fortran
   DEFINITIONS USE_MPI)
 set_target_properties(${TESTO} PROPERTIES LINK_FLAGS "${OpenMP_Fortran_FLAGS}")
-target_link_libraries(${TESTO} GMAO_pFIO ${NETCDF_LIBRARIES})
+#target_link_libraries(${TESTO} MAPL_Shared GMAO_pFIO ${NETCDF_LIBRARIES})
```

Comment:
@weiyuan-jiang If we don't need this line, perhaps we should remove it? Rather than keep it as a comment? Though maybe @tclune should weigh in on the `CMakeLists.txt` as a whole...

Reply:
I think we should remove it.   In a few lines above (ecbuild_add_executable) , the lib is already passed in.

Reply:
removed